### PR TITLE
Update understanding-config-files.md

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: doc/getting-started/understanding-config-files.md [hunk 1]
```diff
@@ -29,114 +29,106 @@ But it is important because it is used to set:
...
+	  "maxLovelaceSupply": 45000000000000000,
+	  "networkMagic": 764824073,
+	  "epochLength": 432000,
+	  "systemStart": "2017-09-23T21:44:51Z",
+	  "slotsPerKESPeriod": 129600,
+	  "slotLength": 1,
+	  "maxKESEvolutions": 62,
+	  "securityParam": 2160
+	}
+```
```
Comment (intricate) [inline doc/getting-started/understanding-config-files.md:105]: This shouldn't be indented at all.

File: doc/getting-started/understanding-config-files.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Tells your node to which nodes in the network it should talk to. A minimal version of this file looks like this:
 
-
+```json
 	{
```
Comment (intricate) [inline doc/getting-started/understanding-config-files.md:8]: Same with the indentation here.

File: doc/getting-started/understanding-config-files.md [hunk 3]
```diff
@@ -29,114 +29,106 @@ But it is important because it is used to set:
...
 | maxBlockBodySize | Maximal block body size |
-| keyMinRefund | The minimum percent refund guarantee |
-| minFeeB | The constant factor for the minimum fee calculation |
-| maxBlockBodySize | Maximal block body size |
-| keyMinRefund | The minimum percent refund guarantee |
 | minFeeB | The constant factor for the minimum fee calculation |
 | eMax | Epoch bound on pool retirement |
 | extraEntropy | Well, extra entropy =) |
-| maxBlockHeaderSize | |
+| maxBlockHeaderSize | Maximal block header size |
```
Comment (intricate) [inline doc/getting-started/understanding-config-files.md:122]:
```suggestion
| maxBlockHeaderSize | Maximum block header size |
```

Just for consistency with some of the other table entries here.

File: doc/getting-started/understanding-config-files.md [hunk 4]
```diff
@@ -158,22 +150,21 @@ __NOTE Due to how the config.json file is generated, fields on the real file are
 
 #### Basic Node Configuration.
 
-First section relates the basic node configuration parameters. Make sure you have to `TPraos`as the protocol, the correct path to the `mainnet-shelley-genesis.json` file, `RequiresMagic`for its use in a testnet.
+First section relates the basic node configuration parameters. Make sure you have `TPraos` as the protocol, the correct path to the `mainnet-shelley-genesis.json` file, and `RequiresMagic`for its use in a testnet.
```
Comment (intricate) [inline doc/getting-started/understanding-config-files.md:153]:
```suggestion
First section relates the basic node configuration parameters. Make sure you have `TPraos` as the protocol, the correct path to the `mainnet-shelley-genesis.json` file, and `RequiresMagic` for its use in a testnet.
```

File: doc/getting-started/understanding-config-files.md [hunk 5]
```diff
@@ -158,22 +150,21 @@ __NOTE Due to how the config.json file is generated, fields on the real file are
...
-	{
 	  "Protocol": "TPraos",
 	  "GenesisFile": "mainnet-shelley-genesis.json",
 	  "RequiresNetworkMagic": "RequiresMagic",
 
-#### Update parameteres
+#### Update parameters
 
-This protocol version number gets used by block producing nodes as part of the system for agreeing on and synchronising protocol updates.You just need to be aware of the latest version supported by the network. You dont need to change anything here.
+This protocol version number gets used by block producing nodes as part of the system for agreeing on and synchronising protocol updates. You just need to be aware of the latest version supported by the network. You dont need to change anything here.
```
Comment (intricate) [inline doc/getting-started/understanding-config-files.md:162]:
```suggestion
This protocol version number gets used by block producing nodes as part of the system for agreeing on and synchronising protocol updates. You just need to be aware of the latest version supported by the network. You don't need to change anything here.
```

File: doc/reference/configuring-a-node-using-yaml.md [hunk 6]
```diff
@@ -21,23 +21,24 @@ This declares the basic node configuration parameters (TPraos as the protocol, t
 
 The basic configuration section might look like this:
 
-```{
-  "Protocol": "TPraos",
-  "GenesisFile": "ff-genesis.json",
-  "RequiresNetworkMagic": "RequiresMagic",
-  ```
+```
+  { "Protocol": "TPraos",
```
Comment (intricate) [inline doc/reference/configuring-a-node-using-yaml.md:25]:
```suggestion
{
  "Protocol": "TPraos",
```

**Review by erikd — APPROVED** [review]

File: doc/reference/configuring-a-node-using-yaml.md [hunk 7]
```diff
@@ -21,23 +21,25 @@ This declares the basic node configuration parameters (TPraos as the protocol, t
 
 The basic configuration section might look like this:
 
-```{
+```
```
Comment (newhoggy) [inline doc/reference/configuring-a-node-using-yaml.md:24]: Needs `json`

**Review by newhoggy — APPROVED** [review]

**Pull request merged**